imports/plugins/included/marketplace/client/templates/shops/shopSelect.js

```diff
@@ -0,0 +1,71 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { ReactiveVar } from "meteor/reactive-var";
+import { Reaction } from "/lib/api";
+import { SellerShops } from "/lib/collections";
+
+Template.shopSelect.onCreated(function () {
+  this.autorun(() => {
+    // watch path change to reset toggle
+    Reaction.Router.watchPathChange();
+  });
+});
+
+Template.shopSelect.helpers({
+  sellerShops() {
+      const currentShopId = Reaction.Router.getParam("shopId") || 0;
+      const selector = {
+        // ignore blank site
+        _id: {
+          $ne: "ddzuN2YPvgvx7rJS5"
```

Comment:
Yeah, this feels hokey.. the only point of the seed data having two shops was to force all tests, UI,etc to always be dealing with two shops in the data.  I solution I'd prefer would be be able add a `enabled` property for the shop, to allow a shop to be disabled, and then filtered here.